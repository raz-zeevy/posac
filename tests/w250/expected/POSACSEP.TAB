 зддддддбдддддддддддбдддддддддддбдддддддддддбддддддддддд©
 Ё ITEM Ё   POLAR   Ё  ACC/ATT  Ё   PROMO   Ё   MODIF   Ё
 цддддддедддддддддбдедддддддддбдедддддддддбдедддддддддбд╢
 Ё   1  Ё  186.67 ЁYЁ  186.67 ЁCЁ  186.67 ЁYЁ  186.67 ЁTЁ
 цддддддедддддддддедедддддддддедедддддддддедедддддддддед╢
 Ё   2  Ё 3166.67 ЁYЁ 1173.33 ЁCЁ  960.00 ЁXЁ  960.00 ЁTЁ
 цддддддедддддддддедедддддддддедедддддддддедедддддддддед╢
 Ё   3  Ё 3186.67 ЁXЁ 2186.67 ЁTЁ 1653.34 ЁYЁ 1653.33 ЁTЁ
 цддддддедддддддддедедддддддддедедддддддддедедддддддддед╢
 Ё   4  Ё     .00 ЁXЁ     .00 ЁCЁ     .00 ЁYЁ     .00 ЁCЁ
 юддддддадддддддддададддддддддададддддддддададддддддддады
