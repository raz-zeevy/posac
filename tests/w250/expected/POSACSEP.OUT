
                                        *******************************************************
                                        *                      POSACSEP                       *
                                        *   A PROGRAM FOR OPTIMALLY PARTITIONING POSAC SPACE  *
                                        *               BY EACH ITEM (VARIABLE)               *
                                        *******************************************************



 ITEM NO.  1
 -----------

 For 0 corners: minimum deviation is   186.67 or  .177 per profile
 Number of different deviant profiles is   2
 Frequency of deviant profiles is   52 or   4.92% of the total

 Dividing line from (   .00, 46.67) to (100.00, 46.67)


 For 1 corners: minimum deviation is   186.67 or  .177 per profile
 Number of different deviant profiles is   1
 Frequency of deviant profiles is   14 or   1.32% of the total

 Dividing line from (   .00, 46.67) to ( 95.45, 46.67)
 Dividing line from ( 95.45, 46.67) to ( 95.45,   .00)


 For 2 corners: minimum deviation is   186.67 or  .177 per profile
 Number of different deviant profiles is   2
 Frequency of deviant profiles is   52 or   4.92% of the total

 Dividing line from (   .00, 46.67) to ( 67.47, 46.67)
 Dividing line from ( 67.47, 46.67) to ( 67.47, 43.00)
 Dividing line from ( 67.47, 43.00) to (100.00, 43.00)


 For 3 corners: minimum deviation is   186.67 or  .177 per profile
 Number of different deviant profiles is   1
 Frequency of deviant profiles is   14 or   1.32% of the total

 Dividing line from (  2.89,100.00) to (  2.89, 46.67)
 Dividing line from (  2.89, 46.67) to ( 96.03, 46.67)
 Dividing line from ( 96.03, 46.67) to ( 96.03, 20.81)
 Dividing line from ( 96.03, 20.81) to (100.00, 20.81)


 ITEM NO.  2
 -----------

 For 0 corners: minimum deviation is  3166.67 or 2.996 per profile
 Number of different deviant profiles is   6
 Frequency of deviant profiles is  207 or  19.58% of the total

 Dividing line from (   .00, 46.67) to (100.00, 46.67)


 For 1 corners: minimum deviation is  1173.33 or 1.110 per profile
 Number of different deviant profiles is   4
 Frequency of deviant profiles is  101 or   9.56% of the total

 Dividing line from (   .00, 66.67) to ( 83.00, 66.67)
 Dividing line from ( 83.00, 66.67) to ( 83.00,   .00)


 For 2 corners: minimum deviation is   960.00 or  .908 per profile
 Number of different deviant profiles is   3
 Frequency of deviant profiles is   85 or   8.04% of the total

 Dividing line from ( 18.16,100.00) to ( 18.16, 66.67)
 Dividing line from ( 18.16, 66.67) to ( 85.16, 66.67)
 Dividing line from ( 85.16, 66.67) to ( 85.16,   .00)


 For 3 corners: minimum deviation is   960.00 or  .908 per profile
 Number of different deviant profiles is   2
 Frequency of deviant profiles is   40 or   3.78% of the total

 Dividing line from ( 17.68,100.00) to ( 17.68, 66.67)
 Dividing line from ( 17.68, 66.67) to ( 82.06, 66.67)
 Dividing line from ( 82.06, 66.67) to ( 82.06,  1.35)
 Dividing line from ( 82.06,  1.35) to (100.00,  1.35)


 ITEM NO.  3
 -----------

 For 0 corners: minimum deviation is  3186.67 or 3.015 per profile
 Number of different deviant profiles is   7
 Frequency of deviant profiles is  163 or  15.42% of the total

 Dividing line from ( 53.33,100.00) to ( 53.33,   .00)


 For 1 corners: minimum deviation is  2186.67 or 2.069 per profile
 Number of different deviant profiles is   4
 Frequency of deviant profiles is  110 or  10.41% of the total

 Dividing line from ( 40.00,100.00) to ( 40.00, 15.21)
 Dividing line from ( 40.00, 15.21) to (100.00, 15.21)


 For 2 corners: minimum deviation is  1653.34 or 1.564 per profile
 Number of different deviant profiles is   5
 Frequency of deviant profiles is  108 or  10.22% of the total

 Dividing line from (   .00, 86.67) to ( 60.00, 86.67)
 Dividing line from ( 60.00, 86.67) to ( 60.00, 17.51)
 Dividing line from ( 60.00, 17.51) to (100.00, 17.51)


 For 3 corners: minimum deviation is  1653.33 or 1.564 per profile
 Number of different deviant profiles is   5
 Frequency of deviant profiles is  124 or  11.73% of the total

 Dividing line from ( 20.00,100.00) to ( 20.00, 75.93)
 Dividing line from ( 20.00, 75.93) to ( 60.00, 75.93)
 Dividing line from ( 60.00, 75.93) to ( 60.00, 18.72)
 Dividing line from ( 60.00, 18.72) to (100.00, 18.72)


 ITEM NO.  4
 -----------

 For 0 corners: minimum deviation is      .00 or  .000 per profile
 Number of different deviant profiles is   0
 Frequency of deviant profiles is    0 or    .00% of the total

 Dividing line from ( 53.03,100.00) to ( 53.03,   .00)


 For 1 corners: minimum deviation is      .00 or  .000 per profile
 Number of different deviant profiles is   0
 Frequency of deviant profiles is    0 or    .00% of the total

 Dividing line from (   .00, 96.00) to ( 47.13, 96.00)
 Dividing line from ( 47.13, 96.00) to ( 47.13,   .00)


 For 2 corners: minimum deviation is      .00 or  .000 per profile
 Number of different deviant profiles is   0
 Frequency of deviant profiles is    0 or    .00% of the total

 Dividing line from (   .00, 94.79) to ( 48.01, 94.79)
 Dividing line from ( 48.01, 94.79) to ( 48.01,  1.70)
 Dividing line from ( 48.01,  1.70) to (100.00,  1.70)


 For 3 corners: minimum deviation is      .00 or  .000 per profile
 Number of different deviant profiles is   0
 Frequency of deviant profiles is    0 or    .00% of the total

 Dividing line from (   .00, 97.22) to ( 47.02, 97.22)
 Dividing line from ( 47.02, 97.22) to ( 47.02, 60.03)
 Dividing line from ( 47.02, 60.03) to ( 51.20, 60.03)
 Dividing line from ( 51.20, 60.03) to ( 51.20,   .00)


